src/components/Popover/usePopoverContentAlignment.tsx

```diff
@@ -0,0 +1,136 @@
+import { PopoverContentProps } from './PopoverContent';
+import { Position, getCollisions } from '@reach/popover';
+
+export const positionBottomLeft = (offset: [number, number]): Position => (
```

Comment:
That's odd and overengineered IMHO.
It also means that we should potentially detect window size changes and rearrange the popover accordingly.
Ideally, I would expect to add a "placeholder element" underneath the popover triggering element and use absolute positioning for the content. I am not also sure if we would actually use the placement options for now.

Reply:
Gotcha, I guess that's the way to go in order to avoid frustration and CSS overrides.